Update/kage_003.txt

```diff
@@ -2302,20 +2302,20 @@ void main()
 //r私が『すごい』といった感嘆には、南井警部が自分よりもまず、部下や同僚たちをどれだけ誇りにしているかを、今の言葉で感じたことも含まれていた。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "私が『すごい』といった感嘆には、南井警部が自分よりもまず、部下や同僚たちをどれだけ誇りにしているかを、今の言葉で感じたことも含まれていた。",
-			NULL, "I also thought it was \"amazing\" how proud Chief Inspector Minai was of her subordinates and colleagues.", Line_Normal);
+			NULL, "Part of what I thought was \"amazing\" was how I could feel how proud Chief Inspector Minai was of her subordinates and colleagues over herself.", Line_Normal);
 	ClearMessage();
 
 //r結果や報酬ではなく、自分の環境や仲間たちを自慢にできることは、本当に幸せだと思う。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "結果や報酬ではなく、自分の環境や仲間たちを自慢にできることは、本当に幸せだと思う。",
-			NULL, "I felt there was true happiness in being able to boast about your comrades and surroundings, rather than your results and awards.", Line_Normal);
+			NULL, "She see truly happy to be able to boast about her own coworkers and environment, rather than her own results and awards.", Line_Normal);
```

Comment:
```suggestion
			NULL, "She seemed truly happy to boast about her coworkers and environment, rather than her own results and awards.", Line_Normal);
```